configs/bi/VT.yaml

```diff
@@ -3,6 +3,6 @@ state: VT
 links:
 - name: primary
   eval: True
-  url: (date.today()).strftime("https://www.healthvermont.gov/sites/default/files/documents/pdf/COVID19-Weekly-Data-Summary-%-m-%-d-%y.pdf")
+  url: (date.today() - timedelta(days=1)).strftime("https://www.healthvermont.gov/sites/default/files/documents/pdf/COVID19-Weekly-Data-Summary-%-m-%-d-%Y.pdf")
```

Comment:
I added it in case they update late in the day after we run the screenshotter. I figured we're better off being a day late and knowing that we'll capture it when they add the report. What do you think?